dashboard/deploy_dashboard_gcloud

```diff
@@ -11,6 +11,30 @@ cd $ROOT/dashboard
 PROJECT=$1
 shift
 
+udmi_state=udmi_state
+udmi_reflect=udmi_reflect
+udmi_target=udmi_target
+udmi_config=udmi_config
+
+append_env_var () {
+   extra_args="$extra_args --set-env-vars $1=$2"
+} 
+
+while (( $# > 0 )) ; do
+    k=$(echo $1 | cut -d'=' -f1)
+    v=$(echo $1 | cut -d'=' -f2)
+
+  case $k in
+    --udmi_state) udmi_state=$v && append_env_var UDMI_STATE $v;;
```

Comment:
I would keep these clauses as simple as possible, e.g., either have "append_env_var" or "udmi_state =" but not both. If the later, then have code elsewhere that converts them to the necessary ENV variable (but as per the other comment I don't understand what the udmi_state variable is for anyway).